lib/MusicBrainz/Server/Controller/Release.pm

```diff
@@ -307,7 +306,7 @@ sub add_cover_art : Chained('load') PathPart('add-cover-art') Edit {
         images => \@artwork,
         mime_types => \@mime_types,
         access_key => DBDefs->COVER_ART_ARCHIVE_ACCESS_KEY // '',
-        cover_art_types_json => $json->encode(
+        cover_art_types_json => $c->json->encode(
```

Comment:
Is `$c->json` a reference to one of those modules? (I can't find it in Catalyst docs.)

Also, how does it know which module to use? By reading `PERL_JSON_BACKEND` variable?